universal-login-sdk/src/core/utils/events.ts

```diff
@@ -2,7 +2,7 @@ import {WalletContractInterface, GnosisSafeInterface} from '@unilogin/contracts'
 import {Log} from 'ethers/providers';
 import {WalletEventArgs, WalletEventType} from '../models/events';
 
```

Comment:
why we don't export it anymore?

Reply:
https://github.com/Microsoft/TypeScript/issues/5711